dev/packages/alpha/mysql-0.0.2/dataset/status/manifest.yml

```diff
@@ -0,0 +1,19 @@
+id: status
+title: MySQL status metrics
+release: ga
+type: metrics
+streams:
+- input: mysql/metrics
+  vars:
+  - name: period
+    type: text
+    title: Period
+    required: true
+    show_user: true
+    default: 10s
+  - name: raw
+    type: bool
+    title: Raw (all fields from "SHOW /*!50002 GLOBAL */ STATUS;")
```

Comment:
Should we put this into the description?

Reply:
Fixed.